docs/developers_guide.md

```diff
@@ -318,3 +318,7 @@ Wed May 24 02:04:40 2023
 +---------------------------------------------------------------------------------------+
 
 ```
+
```

Comment:
I think this section should much higher up in this doc. Also, I don't think it deserves an entire section. just find the appropriate place in this doc

Reply:
There is a logging.md file. I think that's a good place for that, if we are renaming it to LogVerbosity.